src/DOM/Classy/ParentNode.purs

```diff
@@ -1,19 +1,15 @@
 module DOM.Classy.ParentNode where
 
 import Prelude
-
-import Control.Monad.Eff (Eff)
-
-import Data.Maybe (Maybe)
-import Data.Nullable (Nullable)
-
-import DOM (DOM)
-import DOM.Classy.Util (fromAny)
 import DOM.HTML.Types as H
 import DOM.Node.ParentNode as PN
 import DOM.Node.Types as N
-
 import Unsafe.Coerce as U
+import Control.Monad.Eff (Eff)
+import DOM (DOM)
+import DOM.Classy.Util (fromAny)
+import DOM.Node.ParentNode (QuerySelector)
+import Data.Maybe (Maybe)
```

Comment:
hah, all right, pushed it up. i forgot letting psc-ide import stuff jumbles up the imports.